server.js

```diff
@@ -32,23 +43,30 @@ function countString(count, str) {
   return `${count} ${string}`;
 }
 
-// Internal: Accepts a json report object, console logs the results
-// and quits the process with either exit code 1 or 0 depending on whether any
-// tests failed.
-function processReport(resultsJSON) {
-  const { results, fullResults, errorCount, duration } = resultsJSON;
-
-  results.forEach((result, index) => {
-    message = `${index + 1}) ${result['message']}`;
-
-    if (result['passed']) {
-      // Log green test result if test passed
-      console.log(chalk.green(message));
-    } else {
-      // Log red test result if test failed
-      console.log(chalk.red(message));
-    }
-  })
+// Internal: Accepts a test result json object and console.logs the result.
+function logTestResult(testResultJson) {
+  const { message, passed } = testResultJson;
+
+  server.locals.testCount++ ;
+  formattedMessage = `${server.locals.testCount}) ${message}`;
+
+  if (passed) {
+    // Log green test result if test passed
+    console.log(chalk.green(formattedMessage));
+  } else {
+    // Log red test result if test failed
+    console.log(chalk.red(formattedMessage));
+  }
+};
+
+// Internal: Accepts a json report object, console.logs the overall result of
+// the test suite and quits the process with either exit code 1 or 0 depending
+// on whether any tests failed.
+function shutDownServer(reportJson) {
```

Comment:
Not sure about this function name.....

Reply:
maybe something like `finishTesting` ? Because it does more things that just shut down the server, and in the case `dev = true` it doesn't shut it down at all.